x/incentives/module.go

```diff
@@ -187,17 +189,20 @@ func (AppModule) GenerateGenesisState(simState *module.SimulationState) {
 
 // ProposalContents doesn't return any content functions for governance
 // proposals.
+// TODO: Explain this better
 func (AppModule) ProposalContents(simState module.SimulationState) []simtypes.WeightedProposalContent {
 	return nil
 }
 
 // RandomizedParams creates randomized pool-incentives param changes for the
 // simulator.
+// TODO: This should't be nil right?
 func (AppModule) RandomizedParams(r *rand.Rand) []simtypes.ParamChange {
 	return nil
 }
 
 // RegisterStoreDecoder registers a decoder for supply module's types.
+// TODO: This also appears to do nothing
```

Comment:
TODO: This also appears to do nothing

Reply:
idek what this was meant for in ~any simulator design